pkg/virt-controller/watch/vm_test.go

```diff
@@ -1186,6 +1187,334 @@ var _ = Describe("VirtualMachine", func() {
 				})
 			})
 		})
+
+		Context("VM printableStatus", func() {
+
+			table.DescribeTable("should set a Stopped status when VMI doesn't exist", func(running bool, runStrategy v1.VirtualMachineRunStrategy) {
+				vm, _ := DefaultVirtualMachine(running)
+
+				if runStrategy != v1.RunStrategyUnknown {
+					vm.Spec.RunStrategy = &runStrategy
+					vm.Spec.Running = nil
+				}
+
+				addVirtualMachine(vm)
+
+				vmInterface.EXPECT().UpdateStatus(gomock.Any()).Times(1).Do(func(obj interface{}) {
+					objVM := obj.(*v1.VirtualMachine)
+					Expect(objVM.Status.PrintableStatus).To(Equal(v1.VirtualMachineStatusStopped))
+				})
+
+				controller.Execute()
+			},
+
+				table.Entry("running: false", false, v1.RunStrategyUnknown),
+				table.Entry("runStrategy: Halted", false, v1.RunStrategyHalted),
+				table.Entry("running: Manual", false, v1.RunStrategyManual),
+			)
+
+			table.DescribeTable("should set a Stopped status when VMI has stopped", func(phase v1.VirtualMachineInstancePhase, deletionTimestamp bool) {
```

Comment:
I think you shouldn't use a boolean argument, there's hardly ever a motivation for it.

In this case, you can achieve the exact same passing directly a `*Time` struct.

Reply:
Good point. Will change to `*Time`.